test-result-summary-client/src/Dashboard/Widgets/Graph/ODM.jsx

```diff
@@ -80,14 +80,13 @@ export default class ODM extends Component {
 
         // combine results having the same JDK build date
         results.forEach(( t, i ) => {
-            if ( t.buildResult !== "SUCCESS" ) return;
-
+            if ( t.buildResult !== "SUCCESS" || ! Array.isArray(t.aggregateInfo) || t.aggregateInfo.length < 1 || ! t.aggregateInfo[0].jdkDate || ! Array.isArray(t.aggregateInfo[0].metrics) || t.aggregateInfo[0].metrics.length < 1 || ! t.aggregateInfo[0].metrics[0]["value"] || Object.keys(t.aggregateInfo[0].metrics[0]["value"]).length === 0) return;
             // TODO: current code only considers one interation. This needs to be updated

```

Comment:
Currently, we should add support to parse all metrics under one benchmark job in future PR since we're only parsing one hard-coded metric (`Global Throughput`) for now.